bedrock/firefox/templates/firefox/accounts-2019.html

```diff
@@ -0,0 +1,152 @@
+
+    <ul class="c-product-list">
+      <li class="c-product-list-item t-product-firefox">
+        <a href="{{ url('firefox') }}">
+          {# L10n: This is not a complete proper name; "Firefox" is a proper name that shouldn't be translated, but you can choose to translate "browser". Span for formatting. #}
+          <h3 class="c-product-list-title">{{ _('Firefox <span>browser</span>') }}</h3>
+          <p class="c-product-list-desc">{{ _('Travel the internet with protection, on every device.') }}</p>
+        </a>
+      </li>
+      <li class="c-product-list-item t-product-lockwise">
+        <a href="https://lockwise.firefox.com?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=accounts-trailhead&amp;utm_content=product-list-item&amp;entrypoint=mozilla.org-accounts_page" data-link-name="Firefox Lockwise" data-link-type="link">
+          <h3 class="c-product-list-title">Firefox <span>Lockwise</span></h3>
+          <p class="c-product-list-desc">{{ _('Keep your passwords protected and portable.') }}</p>
+        </a>
+      </li>
+      <li class="c-product-list-item t-product-monitor">
+        <a href="https://monitor.firefox.com?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=accounts-trailhead&amp;utm_content=product-list-item&amp;entrypoint=mozilla.org-accounts_page" data-link-name="Firefox Monitor" data-link-type="link">
+          <h3 class="c-product-list-title">Firefox <span>Monitor</span></h3>
+          <p class="c-product-list-desc">{{ _('Get a lookout for data breaches.') }}</p>
+        </a>
+      </li>
+      <li class="c-product-list-item t-product-send">
+        <a href="https://send.firefox.com?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=accounts-trailhead&amp;utm_content=product-list-item&amp;entrypoint=mozilla.org-accounts_page" data-link-name="Firefox Send" data-link-type="link">
+          <h3 class="c-product-list-title">Firefox <span>Send</span></h3>
+          <p class="c-product-list-desc">{{ _('Share large files without prying eyes.') }}</p>
+        </a>
+      </li>
+    </ul>
+
+    <p class="c-accounts-products-tagline">{{ _('And get it all on every device, without feeling trapped in a single operating system.') }}</p>
+  </div>
+</section>
+
+
+<section class="c-accounts-value t-value-respect l-media-left">
+  <div class="c-accounts-value-container">
+    <div class="c-accounts-value-media">
+      {{ high_res_img('firefox/accounts/trailhead/value-respect.jpg', {'alt': '', 'width': '800', 'height': '600'}) }}
+    </div>
+
+    <div class="c-accounts-value-content">
+      <h3 class="c-accounts-value-title">{{ _('Get the respect you deserve.') }}</h3>
+
+      <p>
+      {% trans promise='https://blog.mozilla.org/firefox/firefox-data-privacy-promise/?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=accounts-trailhead&amp;utm_content=accounts-value&amp;utm_term=respect-you-deserve' %}
```

Comment:
Note: this blog post is not yet live, and blocks this page from going to production. I'll consult with MJ on when the post can be published.